src/components/AuthButtonsGroup/AuthButtonsGroup.styled.js

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+
+export const AuthBtnsWrapper = styled.div`
+    display: none;
+
+    @media screen and (min-width: ${props => props.theme.sizes.desk}) {
+        display: inline;
+    }
+`;
+
+export const LoginLink = styled(Link)`
+    background-color: ${props => props.theme.colors.lightBlue};
```

Comment:
I would suggest to use destructuring to get access to `props`, to improve readability and save string size:
```
background-color: ${({theme}) => theme.colors.lightBlue};
```

Reply:
Ok, I will use it.